crates/tx5/examples/mem-echo-stress.rs

```diff
@@ -0,0 +1,92 @@
+//! Mem backend timings. The hope is to have near ideal zero overhead.
+//! At time of writing, you can see we're pretty close to exactly
+//! proportional to node count for a given machine's resources:
+//!
+//! - 1 nodes, 33941 ops, 0.000028 sec/op, in 1.004478s
+//! - 10 nodes, 174041 ops, 0.000056 sec/op, in 1.016977s
+//! - 100 nodes, 175111 ops, 0.000580 sec/op, in 1.018943s
+//! - 1000 nodes, 160266 ops, 0.006361 sec/op, in 1.019859s
+
+use std::sync::Arc;
+use tx5::{backend::*, *};
+
+#[tokio::main(flavor = "multi_thread")]
+async fn main() {
+    let fake_sig = SigUrl::parse("wss://fake.fake").unwrap();
+
+    let config = Arc::new(Config {
+        backend_module: BackendModule::Mem,
+        ..Default::default()
+    });
+
+    let (listen_ep, mut listen_recv) = Endpoint::new(config.clone());
+    let listen_ep = Arc::new(listen_ep);
+    let listen_url = listen_ep.listen(fake_sig.clone()).await.unwrap();
+
+    tokio::task::spawn(async move {
+        while let Some(evt) = listen_recv.recv().await {
+            match evt {
+                EndpointEvent::ListeningAddressOpen { .. } => (),
+                EndpointEvent::Connected { .. } => (),
+                EndpointEvent::Message { peer_url, .. } => {
+                    let listen_ep = listen_ep.clone();
+                    // Don't hold up our recv loop on the response
+                    tokio::task::spawn(async move {
```

Comment:
Is it faster if you create just one task for responses with a channel that sends the response when it receives something from the trigger, and just trigger that channel for every incoming message?

Reply:
It might be, but this is overhead in the test setup, and I was happy with the overall overhead, so don't feel the need to optimize further.